packages/ui/src/components/va-input/components/VaInputWrapper.vue

```diff
@@ -307,8 +307,11 @@ export default defineComponent({
 
   &--labeled {
```

Comment:
not only for `--labeled`

---

But it seems that now the default `va-input` field `width` is `100%` and there is no more this problem

Reply:
> not only for `--labeled`
> 
> But it seems that now the default `va-input` field `width` is `100%` and there is no more this problem

Yeah, didn't mention. Moved to common input content wrapper class.

Problem was reproduced by me yesterday (honestly, can't remember how exactly), moreover, width 100% was put there in 2021, but this is 2022 issue.